templates/EN.html

```diff
@@ -102,6 +102,7 @@
         <span class='tempunit'> &deg;C</span>
       </div>
 
+{#    todo maybe we should remove the air speed selector since the EN-15251 standard does not compensate for elevated air speed    #}
```

Comment:
Would this show up in the actual page? I thought html comments looked like `<!-- I'm a comment-->`. Still think an issue is better for this type of discussion and tracking.

Reply:
done